examples/complete/variables.tf

```diff
@@ -0,0 +1,11 @@
+variable "region" {
+  type        = string
+  description = "AWS region"
+  default     = "us-east-2"
+}
+
+variable "marbot_endpoint_id" {
```

Comment:
why this variable not in `examples/marbot` example?
and what is `marbot` ? :)

Reply:
I think this was due to my error when reorganising files.

The module had this marbot example already, but my opinion is it's not that useful.

@aknysh do you agree we can remove this example?

Marbot is a chat bot service for AWS monitoring from cloudonaut.
https://marbot.io/

Reply:
@joe-niland let's remove that example (it's confusing)